### PR TITLE
fix: `Change password` checkbox alignment

### LINKED ISSUE #7055: [Task]: Fix checkbox alignment
### Task description

The checkbox alignment is broken
image

### Requirements

Fix checkbox aligment

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

Closes #7055 

## Changelog

before:
image

now:
image

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by cpl121 — COMMENTED** [review]
lgtm 💯

**Review by cpl121 — APPROVED** [review]
lgtm 💯

File: packages/desktop/views/dashboard/settings/views/security/ChangePassword.svelte [hunk 1]
```diff
@@ -159,13 +159,8 @@
...
         submitHandler={isPasswordValid}
     />
-    <Checkbox
-        classes="mb-5"
-        label={localize('actions.exportNewStronghold')}
-        bind:checked={exportStrongholdChecked}
-        disabled={busy}
-    />
-    <div class="flex flex-row items-center">
+    <Checkbox label={localize('actions.exportNewStronghold')} bind:checked={exportStrongholdChecked} disabled={busy} />
```
Comment (begonaalvarezd) [inline packages/desktop/views/dashboard/settings/views/security/ChangePassword.svelte:162]: What do you think about improving the file in general and making it use space-y?

```html
<form id="form-change-password" on:submit|preventDefault={changePassword}>
    <Text type={TextType.h4} classes="mb-3">{localize('views.settings.changePassword.title')}</Text>
    <div class="flex flex-col w-full space-y-5">
        <Text type={TextType.p} secondary>{localize('views.settings.changePassword.description')}</Text>
        <PasswordInput
            error={localize(currentPasswordError)}
            bind:value={currentPassword}
            showRevealToggle
            placeholder={localize('general.currentPassword')}
            disabled={busy}
            submitHandler={isPasswordValid}
        />
        <PasswordInput
            error={newPasswordError}
            bind:value={newPassword}
            showRevealToggle
            strengthLevels={4}
            showStrengthLevel
            strength={passwordStrength.score}
            placeholder={localize('general.newPassword')}
            disabled={busy}
            submitHandler={isPasswordValid}
        />
        <PasswordInput
            bind:value={confirmedPassword}
            showRevealToggle
            placeholder={localize('general.confirmNewPassword')}
            disabled={busy}
            submitHandler={isPasswordValid}
        />
        <Checkbox
            label={localize('actions.exportNewStronghold')}
            bind:checked={exportStrongholdChecked}
            disabled={busy}
        />
        <div class="flex flex-row items-center">
            <Button
                size={ButtonSize.Medium}
                disabled={!currentPassword || !newPassword || !confirmedPassword || busy}
                type={HTMLButtonType.Submit}
            >
                {localize('views.settings.changePassword.title')}
            </Button>
            <Spinner {busy} message={localize(changeMessageLocale)} classes="ml-2" />
        </div>
    </div>
</form>
```

not super relevant, just a comment

**Review by marc2332 — APPROVED** [review]
:ok_woman:

**Pull request merged**